typings/wizard.d.ts

```diff
@@ -0,0 +1,59 @@
+/** @format */
+
+import { Composer, Middleware } from './composer'
+import { SceneContextOptions } from './stage'
+import { TelegrafContext } from './context'
+
+export interface WizardContext<TContext extends WizardContextMessageUpdate> {
+  ctx: TContext
+
+  options: SceneContextOptions
```

Comment:
https://github.com/telegraf/telegraf/blob/f5bb973015ec637cde26aeabf4c530926df1efe5/scenes/wizard/context.js#L2-L7 doesn't seem to have `options`.